extensions/cli/landsat8/src/main/java/mil/nga/giat/geowave/format/landsat8/RasterIngestRunner.java

```diff
@@ -294,6 +297,45 @@ protected BandData getBandData(
 				geotiffFile);
 	}
 
+	private CoordinateReferenceSystem getIndexCRS(
+			PrimaryIndex[] indices ) {
+
+		CoordinateReferenceSystem indexCrs = null;
+
+		for (PrimaryIndex primaryindx : indices) {
+
+			// for first iteration
+			if (indexCrs == null) {
+				if (primaryindx.getIndexModel() instanceof CustomCrsIndexModel) {
```

Comment:
this whole method could probably be in geometryutils, and this section could reuse the GeometryUtils.getCRS(index) method